source/authentication/custom-jwt.txt

```diff
@@ -801,55 +801,22 @@ Examples
 For code examples that demonstrate how to register and log in using
 Custom JWT authentication, see the documentation for the Realm SDKs:
 
-.. tabs-realm-sdks::
-
-   .. tab::
-      :tabid: android
-
-      To register or log in a Custom JWT user from the Android Client SDK, see the
-      :ref:`Java SDK guide to Custom JWT authentication <java-login-custom-jwt>`.
-
-   .. tab::
-      :tabid: ios
-
-      To register or log in a Custom JWT user from the iOS Client SDK, see the
-      :ref:`Swift SDK guide to Custom JWT authentication <ios-login-custom-jwt>`.
-
-   .. tab::
-      :tabid: web
-
-      To register or log in a Custom JWT user from the Web Client SDK, see the
-      :ref:`Web SDK guide to Custom JWT authentication <web-login-custom-jwt>`.
-
-   .. tab::
-      :tabid: node
-      
-      To register or log in a Custom JWT user from the Node Client SDK, see the
-      :ref:`Node SDK guide to Custom JWT authentication <node-login-custom-jwt>`.
-
-   .. tab::
-      :tabid: react-native
-
-      To register or log in a Custom JWT user from the React Native Client SDK, see the
-      :ref:`React Native SDK guide to Custom JWT authentication <react-native-login-custom-jwt>`.
-
-   .. tab::
-      :tabid: dotnet
-
-      To register or log in a Custom JWT user from the .NET Client SDK, see the
-      :ref:`.NET SDK guide to Custom JWT authentication <dotnet-login-custom-jwt>`.
-
-   .. tab::
-      :tabid: kotlin
-
-      To register or log in a Custom JWT user from the Kotlin Client SDK, see the
-      :ref:`Kotlin SDK guide to Custom JWT authentication <kotlin-login-jwt>`.
-
-   .. tab::
-      :tabid: flutter
-
-      To register or log in a Custom JWT user from the Flutter Client SDK, see the
-      :ref:`Flutter SDK guide to Custom JWT authentication <flutter-login-custom-jwt>`.
+- To register or log in a Custom JWT user from the Flutter SDK, see the
```

Comment:
The repetition was hidden in the tab view, here it is very obvious and kinda distracting. Maybe we should just have the link with one preamble saying "To register ... see the corresponding SDK documentation:"